src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,58 @@
+package step2;
+
+import step2.core.*;
+import step2.util.StringUtils;
+
+import static step2.common.ArithmethicConst.*;
+import static step2.common.ExceptionMessage.*;
+
+public class Calculator {
+
+    public int process(String inputVal){
+
+        String[] splitInputValBySpace = StringUtils.splitInputValBySpace(inputVal);
+        validationInputValue(splitInputValBySpace);
+
+        int resultValue = Integer.parseInt(splitInputValBySpace[ZERO]);
+
+        int arithmethicCount = getArithmethicCount(splitInputValBySpace.length);
+
+        for(int i=0; i< arithmethicCount; i++){
```

Comment:
등호 및 기타 연산자의 앞 뒤에 한 칸씩의 공백을 두었으면 좋겠습니다. _단항 연산자 및 세미콜론은 제외입니다._
```suggestion
        for(int i = 0; i < arithmethicCount; i++){

Reply:
등호 및 기타연산자에 공백 추가하는 버릇을 익히도록하겠습니다